charts/generic-prometheus-alerts/templates/aws-sns-alerts.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.snsAlertsTopicNames -}}
+{{- $targetApplication := required "A value for targetApplication must be set" .Values.targetApplication }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ $targetApplication }}-sns
+  labels:
+    {{- include "generic-prometheus-alerts.labels" . | nindent 4 }}
+spec:
+  groups:
+    - name: {{ $targetApplication }}-sns
+      rules:
+{{- range $.Values.snsAlertsTopicNames }}
+        - alert: SNS-no-messages-published
```

Comment:
Sorry for the slow review (I was off yesterday) - but you should probably include the topic name in the name of the alert.  If a user wants to cover more than one topic I think the name should be unique otherwise it can get a bit odd in the reporting of the alerts as you'll have multiple alerts firing of the same name - they'll have to dig into the detail to figure out which topic is having issues.